### PR TITLE
Problem: formal v0.46.5 cosmos-sdk release is not used

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- update dependency, should be non-breaking for cronos

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	cosmossdk.io/math v1.0.0-beta.3
 	github.com/armon/go-metrics v0.4.1
-	github.com/cosmos/cosmos-sdk v0.46.5-0.20221111074045-68e54fa5b868
+	github.com/cosmos/cosmos-sdk v0.46.6
```
Comment (thomas-nguy) [inline go.mod:8]: can you post the diff? 🙏

Comment (tomtau) [reply go.mod:8]: https://github.com/cosmos/cosmos-sdk/compare/v0.46.5...v0.46.6

Comment (thomas-nguy) [reply go.mod:8]: should be all the commit from `68e54fa5b868` to last right?
https://github.com/cosmos/cosmos-sdk/commits/v0.46.6


anyways I can see a breaking change in group module but I assume it does not affect us

**Review by thomas-nguy — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (tomtau) [reply go.mod:8]: the gov fix potentially affects us / is breaking, I think, but I don't know if there were any new gov proposals executed on the testnet?

Comment (tomtau) [reply go.mod:8]: oh yes, it's from the commit: https://github.com/cosmos/cosmos-sdk/compare/68e54fa5b868...v0.46.6

Comment (yihuang) [reply go.mod:8]: I think the gov change only affect us if we had a proposal execution returned an error.

**Pull request merged**